executor/join.go

```diff
@@ -516,7 +616,22 @@ func (e *HashJoinExec) buildHashTableForList(buildSideResultCh <-chan *chunk.Chu
 		if e.finished.Load().(bool) {
 			return nil
 		}
-		err := e.rowContainer.PutChunk(chk)
+		if e.useOuterToBuild {
+			var bitMap = bitmap.NewConcurrentBitmap(chk.NumRows())
+			e.outerMatchedStatus = append(e.outerMatchedStatus, bitMap)
+			e.memTracker.Consume(bitMap.BytesConsumed())
+			if len(e.probeSideFilter) > 0 {
```

Comment:
It seems that `probeSideFilter` should be renamed to `outerFilter` back.

Reply:
add a `buildSideFilter` for this case?

Reply:
> It seems that `probeSideFilter` should be renamed to `outerFilter` back.

add a `buildSideFilter` for this case?

Reply:
Maybe we do not need buildSideFilter. We only need the `outerFilter`.

Reply:
ok, I rename it.